controllers/oauthAuthentication.js

```diff
@@ -28,14 +35,86 @@ export const oauthLoginAuthorize = (req, res) => {
  * Make every http call to the different API endpoints.
  */
 export const oauthLoginCallback = async (req, res, next) => {
-  // check if the mandatory Authorization code is there
-  if (!req.query.code) {
-    return res.sendStatus(400);
+  /**
+   * OpenID Connect standard errors
+   * @see @link{https://www.rfc-editor.org/rfc/rfc6749.html#section-4.1.2.1}
+   *
+   * l'idée ici présente est de vous montrer un cas de figure: le traitement d'un retour
```

Comment:
putting comment in french maybe a good idea, but we should preserve consistency first then decide which language to use.

Put this in english for now.